study.md

```diff
@@ -54,7 +80,7 @@ already wrote and include the information about Dave's fish.
 Using the JSON your wrote above, can you write a method on the JSON?
 
 ```json
-<!-- your answer here -->
+var jsonString = JSON.stringify(Jason);
```

Comment:
nice, I was looking for a "no" here because you can't have a function in JSON, but I understand what you were going for.  I think you were looking for this thought 

JSON.parse(Jason)